query-engine/connectors/query-connector/src/interface.rs

```diff
@@ -12,6 +12,15 @@ pub trait Connector {
     /// Returns the name of the connector.
     fn name(&self) -> &'static str;
 
+    /// The database system name, as per the OTEL spec.
+    /// In most cases, this is the same as the connector's name.
+    /// Reference:
+    /// - https://opentelemetry.io/docs/specs/semconv/database/sql/
+    /// - https://opentelemetry.io/docs/specs/semconv/database/mongodb/
+    fn system_name(&self) -> &'static str {
```

Comment:
```suggestion
    fn db_system_name(&self) -> &'static str {
```

Reply:
Actually let's just remove this method. The existing `name` method above is only used for adding the attributes to the tracing spans and to return the database name in the `/server_info` HTTP handler in the binary engine; in other words, it is exactly what we need. Let's double check with the features team (https://prisma-company.slack.com/archives/C04EPL7N2KV/p1729072567605489) and just make the `name` method contain a conditional in the postgres connector.